tgui/packages/tgui/interfaces/MixerMachine.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author mloccy (https://github.com/mloccy)
+ * @license ISC
+ */
+
+import { useBackend } from '../backend';
+import { Box, Button, Flex, Section } from '../components';
+import { Window } from '../layouts';
+
+export const MixerItem = (props, context) => {
+  const { act } = useBackend(context);
+
+  const { mixerItem, working } = props;
+
+  return (
+    <Flex>
+      <Flex.Item nowrap key={mixerItem.name}>
```

Comment:
Not sure what the purpose of this `Flex` with a single `Flex.Item` is? Flex is a layout component so is used to arrange multiple `Flex.Item` children.

Reply:
I was trying things, this was a remenent of something that didn't work, removed.